egs/wsj/s5/utils/data/perturb_data_dir_volume.sh

```diff
@@ -7,6 +7,17 @@
 # the wav.scp to perturb the volume (typically useful for training data when
 # using systems that don't have cepstral mean normalization).
 
```

Comment:
how much testing have you done with this?  Can you please test it with directories that have pipes and those that have raw wav files, and make sure it doesn't break in a very obvious way?